livestyled/models/user.py

```diff
@@ -292,14 +292,16 @@ def placeholder(
 class UserAlias:
     def __init__(
             self,
+            id: int,
             user_alias_type,
-            user,
             value,
-            updated_at: datetime,
-            created_at: datetime
+            user: str or None = None,
+            updated_at: datetime or None = None,
+            created_at: datetime or None = None
     ):
+        self.id = id
         self.user_alias_type = User.placeholder(int(user_alias_type))
-        self.user = User.placeholder(int(user))
+        self.user = User.placeholder(int(user)) if user else None
```

Comment:
How can the user alias exist without a user?

Reply:
if we make request `/v4/user_manager/users/{id}`
we get in request ```"userAliases": [
                {
                    "@id": "/user_management/user_aliases/510",
                    "@type": "UserAlias",
                    "id": 510,
                    "userAliasType": "/user_management/user_alias_types/4",
                    "value": "test1234"
                }
            ]```

Reply:
so why do we even need the user property at all?

Reply:
removed, as we don't have any `get_user_alias` method for now